plugins/package-managers/node/src/main/kotlin/utils/NpmSupport.kt

```diff
@@ -69,18 +69,16 @@ internal fun expandNpmShortcutUrl(url: String): String {
 /**
  * Do various replacements in [downloadUrl]. Return the transformed URL.
  */
-internal fun fixNpmDownloadUrl(downloadUrl: String): String {
+internal fun String.fixNpmDownloadUrl(): String =
```

Comment:
Could you detail the motivation for this refactoring a bit in the commit message? I mean, is the code now really more readable? Personally, I started to avoid using extensions functions on primitive types a bit.

Reply:
I find this reads nicer: https://github.com/oss-review-toolkit/ort/pull/9122/commits/5acea97963a06a1e5977cabc2f559ea0f34667f4#diff-42aedc605f6889312d50555012975d3e352996ce23727883d2729757acaaee78L583-R583